pkg/controller.v1/common/job_controller.go

```diff
@@ -66,8 +66,8 @@ type JobController struct {
 	// KubeClientSet is a standard kubernetes clientset.
 	KubeClientSet kubeclientset.Interface
 
-	// KubeBatchClientSet is a standard kube-batch clientset.
-	KubeBatchClientSet kubebatchclient.Interface
```

Comment:
I think my question is, do we want to have some backward compatibility or do we want to support different backends  in case some one want to use `kube-batch` instead of `valcano`?

Reply:
Stand on my own point, as volcano is more actively maintained, it is better to migrate to it in the next release

Reply:
How much efforts would there be to support multiple backends? Would it be possible to make it pluggable?

Reply:
One way is to update `NewJobController`, and add a new argument to indicate whether to use volcano or kube-batch. Then it will require all the operators to change. I think it is not worth doing that.